examples/sumologic.yaml

```diff
@@ -17,7 +17,7 @@ extensions:
   ## The File Storage extension can persist state to the local file system
   ## ref: https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/extension/storage/filestorage
   file_storage:
-    directory: ${FILE_STORAGE:-/var/lib/otelcol-sumo/file_storage}
+    directory: ${SUMOLOGIC_FILE_STORAGE:-/var/lib/otelcol-sumo/file_storage}
```

Comment:
Where is this variable referenced?

Reply:
It will be referenced in documentation only